ios/FluentUI/Core/Theme/Tokens/AliasTokens.swift

```diff
@@ -69,6 +88,50 @@ public final class AliasTokens {
         case .brandDisabled:
             return DynamicColor(light: strongSelf.globalTokens.neutralColors[.grey74],
                                 dark: strongSelf.globalTokens.neutralColors[.grey36])
+        // New foreground colors
+        case .foreground1:
+            return DynamicColor(light: strongSelf.globalTokens.neutralColors[.grey14],
+                                dark: strongSelf.globalTokens.neutralColors[.white])
+        case .foreground1Colorful:
+            return DynamicColor(light: strongSelf.globalTokens.neutralColors[.white])
+        case .foreground2:
+            return DynamicColor(light: strongSelf.globalTokens.neutralColors[.grey38],
+                                dark: strongSelf.globalTokens.neutralColors[.grey84])
+        case .foreground2Colorful:
+            return DynamicColor(light: strongSelf.globalTokens.brandColors[.comm160].light)
+        case .foreground3:
+            return DynamicColor(light: strongSelf.globalTokens.neutralColors[.grey50],
+                                dark: strongSelf.globalTokens.neutralColors[.grey68])
+        case .foreground3Colorful:
+            return DynamicColor(light: strongSelf.globalTokens.brandColors[.comm150].light)
+        case .foregroundDisabled1:
+            return DynamicColor(light: strongSelf.globalTokens.neutralColors[.grey74],
+                                dark: strongSelf.globalTokens.neutralColors[.grey36])
+        case .foregroundDisabled1Colorful:
+            return DynamicColor(light: strongSelf.globalTokens.brandColors[.comm90].light)
+        case .foregroundDisabled2:
+            return DynamicColor(light: strongSelf.globalTokens.neutralColors[.white],
+                                dark: strongSelf.globalTokens.neutralColors[.grey24])
+        case .foregroundContrast:
+            return DynamicColor(light: strongSelf.globalTokens.neutralColors[.black],
+                                dark: strongSelf.globalTokens.neutralColors[.black])
+        case .foregroundContrastColorful:
+            return DynamicColor(light: strongSelf.globalTokens.brandColors[.comm70].light)
+        case .foregroundOnColor:
+            return DynamicColor(light: strongSelf.globalTokens.neutralColors[.white],
+                                dark: strongSelf.globalTokens.neutralColors[.black])
+        case .foregroundOnColorColorful:
+            return DynamicColor(light: strongSelf.globalTokens.brandColors[.comm80].light)
+        case .foregroundInverted1:
+            return DynamicColor(light: strongSelf.globalTokens.neutralColors[.white],
```

Comment:
If the light and dark variations are the same, you can specify only the light variation as it's the only one required.